systems/plants/@RigidBodyManipulator/forwardKin.m

```diff
@@ -1,4 +1,4 @@
-function [x,J,dJ] = forwardKin(obj,kinsol,body_or_frame_ind,pts,rotation_type)
+function [x,J,dJ] = forwardKin(obj,kinsol,body_or_frame_ind,pts,rotation_type, base_or_frame_ind)
```

Comment:
Missed this when I was looking through earlier. `base_or_frame_ind` is something I've wished we had! Kudos!